games-action/minetest_game/minetest_game-5.0.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
Why don't you commit with repoman to update this automatically to its correct form :)